### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by smacker**
Fixes https://github.com/src-d/code-annotation/issues/11

### CODE REVIEW

**Review by carlosms — APPROVED** [review]
Works in my machine

**Review by bzz — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -4,7 +4,16 @@ In order to evaluate quality of ML models, as well as to create “ImageNet for
...
+
+```bash
+docker build -t code-annotation .
+docker run --rm -p 8080:8080 code-annotation
 ```
+
+### Non-docker
+
+```bash
 go get <here will be path>
```
Comment (dpordomingo) [inline README.md:17]: why is it needed the placeholder?
It could be `go get github.com/src-d/code-annotation/...`

File: README.md [hunk 2]
```diff
@@ -4,7 +4,16 @@ In order to evaluate quality of ML models, as well as to create “ImageNet for
...
 cd $GOPATH/<here will be path>
```
Comment (dpordomingo) [inline README.md:18]: placeholder is not needed, is it?
It could be: `cd $GOPATH/src/github.com/src-d/code-annotation`

Comment (bzz) [reply README.md:18]: Both can be handled under https://github.com/src-d/code-annotation/issues/17

Comment (dpordomingo) [reply README.md:18]: great!
added it to the issue desc :D

**Pull request merged**